httpclient/httpclient.go

```diff
@@ -269,7 +274,14 @@ func (h *http2Conn) Write(p []byte) (n int, err error) {
 }
 
 func (h *http2Conn) Close() error {
-	h.out.Close()
 	h.in.Close()
-	return h.Conn.Close()
+	return h.out.Close()
+}
+
+func (h *http2Conn) CloseWrite() error {
+	return h.out.Close()
```

Comment:
This looks backwards, given that `h.out` is a `Reader`, not a `Writer`...

Reply:
Fixed